ambry-frontend/src/main/java/com.github.ambry.frontend/AmbrySecurityService.java

```diff
@@ -260,4 +265,25 @@ private void setBlobPropertiesHeaders(BlobProperties blobProperties, RestRespons
       restResponseChannel.setHeader(RestUtils.Headers.OWNER_ID, blobProperties.getOwnerId());
     }
   }
+
+  /**
+   * Adds the account and container details to the response headers.
```

Comment:
Document that this adds it only when the request has been annotated and the account is not UNKNOWN.